ui/desktop/app/controllers/scopes/scope/projects/settings/index.js

```diff
@@ -18,4 +18,12 @@ export default class ScopesScopeProjectsSettingsIndexController extends Controll
   toggleTheme({ target: { value: theme } }) {
     return this.application.toggleTheme(theme);
   }
+
+  /**
+   * Call Application controller's checkForSessionsRunning method
+   */
+  @action
+  checkForSessionsRunning() {
+    return this.application.checkForSessionsRunning();
+  }
```

Comment:
It's not necessary to create a wrapper function to call the application function. `this.application.checkForSessionRunning` can be passed in the template directly for simplicity.